scripts/generate_cert.sh

```diff
@@ -14,7 +14,7 @@ sudo apt-get update
 sudo apt-get install python-certbot-nginx
 
 # Get certificates (might need sudo)
-certbot certonly --nginx -d "${DOMAIN}" -d "www.${DOMAIN}" --email "${EMAIL}" --agree-tos --redirect
+certbot certonly --nginx -d "${DOMAIN}" -d "www.${DOMAIN}"--email "${EMAIL}" --agree-tos --redirect
```

Comment:
looks like there is a space missing here!

Reply:
Grr.... fixing!